metadata.json

```diff
@@ -90,10 +90,10 @@
   ],
   "description": "Manage SSH",
   "dependencies": [
-    {"name":"puppetlabs/stdlib","version_requirement":">= 4.6.0 < 6.0.0"},
-    {"name":"puppetlabs/concat","version_requirement":">= 2.0.0 < 6.0.0"},
+    {"name":"puppetlabs/stdlib","version_requirement":">= 4.6.0 < 6.4.0"},
```

Comment:
I think you mean `< 7.0.0` as these projects all use semantic versioning.

Reply:
similar for the other entries.

Reply:
Updated version dependencies to account for semantic versioning.